modules/vstudio/_preload.lua

```diff
@@ -152,23 +152,6 @@
 		},
 	}
 
-	p.api.register {   -- DEPRECATED 2019-10-21
```

Comment:
Curious why Git thinks these lines were deleted, rather than just inserting new lines above it. Can you check to make sure your line endings are LF (Unix) on the commit?

Reply:
The line endings are correct, it's because I did some spring cleaning and moved the deprecated fields downwards below the non-deprecated ones to be in line how the fields are arranged elsewhere in the project.